components/appbar/appearance.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Appearance
+page_title: AppBar Appearance
+description: Appearance settings of the AppBar for Blazor.
```

Comment:
Use detailed descriptions with a length of around 150 symbols. This is more like a placeholder.

Reply:
It is not, do you have a suggestion for a better description? The feature itself is rather small